client.go

```diff
@@ -190,6 +190,7 @@ func (c *Client) do(out interface{}, req *http.Request, urlPath string) error {
 	if err != nil {
 		return errors.Wrapf(err, "HTTP Read error on response for %s", urlPath)
 	}
+	c.debugLog(fmt.Sprintf("raw response: %s", string(b)))
```

Comment:
```suggestion
	c.debugLog(fmt.Sprintf("[targetprocess] raw response: %s", string(b)))
```

Reply:
Making this suggestion here with the knowledge that we should go back and put this in to all debugLog messages. (issue #17 opened)

I was inconsistent with it in the initial creation.